astropy/utils/iers/iers.py

```diff
@@ -983,8 +983,8 @@ def auto_open(cls, files=None):
         that expires more than 180 - `~astropy.utils.iers.Conf.auto_max_age`
         after the present.
         """
-        good_enough = cls._today() + TimeDelta(180-_none_to_float(conf.auto_max_age),
-                                               format='jd')
+        offset = 180 - (30 if conf.auto_max_age is None else conf.auto_max_age)
+        good_enough = cls._today() + TimeDelta(180 - offset, format='jd')
```

Comment:
Ah, a little mistake: this should be `TimeDelta(offset, format='jd')`, otherwise we do the `180-` twice!

Reply:
Good catch. Thanks!